CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ if(POLICY CMP0051)
   cmake_policy(SET CMP0051 NEW)
 endif()
 
+if(POLICY CMP0054)
```

Comment:
Im not sure I understand this.  Why would this change suddenly become a problem?  We have been using the new policy, so it would have broken if it wasn't compatible atm, right?